monitor/plugins/spark_progress.py

```diff
@@ -88,37 +89,32 @@ def _publish_measurement(self, job_request):
                 ref_value = (elapsed_time / self.job_expected_time)
 
                 # Error
-                error = job_progress - ref_value
-                
+                if self.job_expected_time < 0.0:
```

Comment:
I think you should treat this case different. What if remaining time is zero? self.job_expected_time is calculated as zero and then divide by zero exception is raised in line 89.

Reply:
@armstrongmsg What is your suggestion?

Reply:
line 75
if remaining_time <= 0:
      job_expected_time = 0
else:
      ...

line 89
if job_expected_time <= 0:
      error = 1.0
else:
       ref_value = ...
       error = job_progress ...